sale_delivery_rate/demo/delivery_carrier.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+    Copyright 2017 LasLabs Inc.
+    License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+-->
+
+<odoo>
+    <record id="demo_carrier_1" model="delivery.carrier">
```

Comment:
I've found that it can be kind of a pain to figure out whether an XML ID corresponds to a demo record or a regular one, so I've been trying to make that clear.